copter/source/docs/traditional-helicopter-rsc-setup.rst

```diff
@@ -4,9 +4,13 @@
 Rotor Speed Control Setup
 =========================
 
+Unlike other vehicles, the throttle RC input channel does not directly control the throttle in what are usually called "non-auto-throttle controlled modes", like STABILIZE or ACRO. Instead it controls the Collective for the helicopter.
```

Comment:
there really isn't a reference anywhere in Copter with a nomenclature for modes where AP controls the thrust and not the pilot..so I just used what was used in Plane..in Plane they are referred to as auto-throttle modes....an alternative I could tink of in Copter would be "altitude controlled modes" since there is no "gliding"....will change to that...okay?